apps/aecore/src/aec_parent_connector.erl

```diff
@@ -100,7 +123,10 @@ handle_info(check_parent, #state{parent_nodes = ParentNodes,
             %% - Call the smart contract to elect new leader.
             %% - Notify conductor of new status
     end,
-    erlang:send_after(State#state.fetch_interval, self(), check_parent),
+    if is_integer(FetchInterval) ->
```

Comment:
Is it possible that `FetchInverval` becomes non-integer? If so - does it make sense silently not scanning the parent chain?

Reply:
I could probably ifdef(TEST). out the place that could happen. For test purposes it's possible to start the process without timed checks. Then test code can trigger parent chain checks at will, and we can have deterministic testing